docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -170,12 +189,34 @@
    "outputs": [],
    "source": [
     "from ess.reflectometry.tools import scale_reflectivity_curves_to_overlap\n",
-    "results_scaled = dict(zip(\n",
-    "    results.keys(),\n",
-    "    scale_reflectivity_curves_to_overlap(results.values()),\n",
-    "    strict=True\n",
-    "))\n",
-    "sc.plot(results_scaled, norm='log', vmin=1e-5)"
+    "\n",
+    "scaled_reflectivity_curves = scale_reflectivity_curves_to_overlap(reflectivity.values())\n",
+    "\n",
+    "sc.plot(dict(zip(reflectivity.keys(), scaled_reflectivity_curves, strict=True)), norm='log', vmin=1e-5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you don't want to apply the scaling the curves but only compute the scale factors that is done by passing the `return_scaling_factors` flag to the routine:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scale_factors = scale_reflectivity_curves_to_overlap(reflectivity.values(), return_scaling_factors=True)\n",
```

Comment:
I know one case when it's useful to get the scale factors and not only the scaled curves.
The case is if you want to scale the `ReflectivityData` type (that is reflectivity as a function of `z` and `\lambda` instead of as a function of `Q`).

Reply:
You can't computed the scale factors directly from `ReflectivityData` (or, you can, but it's cumbersome). So what you should do instead is compute the scale factors using `ReflectivityOverQ` and then multiply each `ReflectivityData` with the corresponding scale factor.

Reply:
We could. But I'm not sure I agree that's a better interface. There are plenty of functions in, for example numpy and scipy, that have similar interfaces. 

Why do you think it's better to have a separerate function?

Here are some drawbacks of making them be separate functions:
* Double amount of docstrings that needs to be kept in sync.
* Less concise documentation, making it more difficult for users to find what they need.
* We would have two very similar functions with close to identical names and signatures. That makes it easy to mix them up.

Reply:
One option is to always return two outputs, the scaled curves and the scaling factors. If we're only interested in the scaled curves we can ignore the second output and vice versa.

The advantage then is that we only need one function and we don't need the keyword argument.

Reply:
> There are plenty of functions in, for example numpy and scipy, that have similar interfaces with boolean arguments controlling what is returned.

Yes, and I don't like them. I think it's very unintuitive for a function to return something completely different depending on an argument.

> One option is to always return two outputs, the scaled curves and the scaling factors.

I like that better.

Reply:
I'll create an issue for this in essreflectometry